.gitmodules

```diff
@@ -2,3 +2,6 @@
 	path = ethcore/res/ethereum/tests
 	url = https://github.com/ethereum/tests.git
 	branch = develop
+[submodule "nsis/win-build"]
```

Comment:
Submodule is not the best way to do it. We don't to push a dll to all users who do a recursive clone.
why not just add something like 
`git archive --remote=https://github.com/ethcore/win-build.git HEAD SimpleFC.dll`
to `appveyor.yml`?